src/secp.rs

```diff
@@ -30,11 +30,13 @@ pub enum TweakKind {
 }
 
 #[derive(Clone, Debug, Eq)]
+#[derive(Default)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Eq, Default)]
```

NIT: you can merge this with the above